addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -130,14 +130,23 @@ odoo.define('website.s_website_form', function (require) {
             if ($dataFor.length || !_.isEmpty(this.preFillValues)) {
                 var dataForValues = $dataFor.length ? JSON.parse($dataFor.data('values').replace('False', '""').replace('None', '""').replace(/'/g, '"')) : [];
                 var fields = _.pluck(this.$target.serializeArray(), 'name');
+                // All types of inputs do not have a value property (eg:hidden),
```

Comment:
Commit message: Why do you not add the full github link as a reference..?
![image](https://user-images.githubusercontent.com/10338094/157030996-6cd2b5da-b632-4788-a922-39d4d92dca46.png)

That's the main point of making references instead of adding the link directly inside the text: being able to see the full link outside of github (git log) and on a dedicated line since the links are long. So in general, replace that by the full link.

However, here it is worse, you link does not actually redirect to anything (maybe since you use a small version of the hash it now conflicts? Even more reason to put the full link): 
![image](https://user-images.githubusercontent.com/10338094/157031219-8ad148b9-0aae-465d-a72d-b13108f890ad.png)